### DISCUSSION TITLE
Autocomplete: add hideOnSelect property

### DISCUSSION THREAD

**Discussion by bogdan0083** [primefaces/community#1501] [PrimeVue]
Right now it's impossible to prevent listbox from hiding when option is selected. I want to be able to leave it open when option is selected and hide it on `escape` keydown or when clicked outside. Here's an example gif of new property (just a prototype):

![Mar-23-2024 14-36-50](https://github.com/primefaces/community/assets/7077307/5bb4cd62-478e-45e3-ba31-06a33de87817)

We can introduce `hideOnSelect` prop which will be `true` by default. But when it's set to `false` the listbox will not be hidden on option select. 

I can send PR if you interested in this feature.

**Comment by graviteski** [comment]
@bogdan0083 This is exactly the functionality that we are looking for. When dealing with many multiple selections from a long list of items, this will save our users many clicks and a great deal of frustration if we can prevent the listbox from hiding between clicks. 

Very keen to have this feature made available.

**Comment by bogdan0083** [comment]
Hi @graviteski ! I finally found a time to implement this feature: https://github.com/primefaces/primevue/pull/5665

Please let me know if something can be improved in terms of code or documentation.
